modules/structs/pull_review.go

```diff
@@ -90,3 +90,20 @@ type SubmitPullReviewOptions struct {
 	Event ReviewStateType `json:"event"`
 	Body  string          `json:"body"`
 }
+
+// PullReviewRequestOptions are options to add or remove pull review request
+type PullReviewRequestOptions struct {
+	Reviewers []string `json:"reviewers"`
+}
+
+// PullReviewRequestErr are error result for add or remove pull review request
+type PullReviewRequestErr struct {
+	Reviewer string `json:"reviewer"`
+	Error    string `json:"error"`
+}
+
+// PullReviewRequestResult results for add or remove pull review request
```

Comment:
is this needed or can this be handled as usual

Reply:
I think it's necessary, Because It can request more than one reviewers by this api, So I think should response results for all reviewer by a list.

Reply:
example result:
```
{
  "Successes": [
    {
      "id": 11,
      "user": {
        "id": 0,
        "login": "mmm",
        "full_name": "",
        "email": "xjy@fchcj.com",
        "avatar_url": "http://192.168.1.5:3000/user/avatar/mmm/-1",
        "language": "",
        "is_admin": false,
        "last_login": "0001-01-01T00:00:00Z",
        "created": "2020-05-07T15:30:21+08:00",
        "username": "mmm"
      },
      "state": "REQUEST_REVIEW",
      "body": "",
      "commit_id": "",
      "stale": false,
      "official": true,
      "comments_count": 0,
      "submitted_at": "2020-05-12T15:42:32+08:00",
      "html_url": "",
      "pull_request_url": "http://192.168.1.5:3000/testg/test_repo/pulls/5"
    }
  ],
  "Failures": [
    {
      "reviewer": "123d",
      "error": "user does not exist [uid: 0, name: 123d, keyid: 0]"
    }
  ]
}
Response 
```